### PR TITLE
[Facebook] Fix grabbing email from login

### LINKED ISSUE #1991: remove weird confirmation screen that occurs after connecting with facebook
I don't think this screen is necessary at all, I'm not sure why we are showing it. I think the user can just continue into the app after they've authorized facebook. 

image

### PR DESCRIPTION
**Pull request by mennenia**
Had an aha moment on the bus, which ended up making this a much simpler change. 

- Our use of the GraphAPI path was out of date, hence we didn't grab any email info from the facebook profile (this includes versions prior to 3.0.0). I've now fixed that in the right place.
- Considering we are now using the correct syntax and we wanted to get rid of that email confirmation screen (it would now pre-populate them), I've deleted that class (`AROnboardingMoreInfoViewController`) - it wasn't being used elsewhere.

I think there are still further improvements we can make to our use of facebook login: #2004

This, however, fixes #1991.

### CODE REVIEW

**Comment by mennenia** [comment]
Tests passed on circle, but there was an issue with a lib for danger? I'm triggered a rebuild.

**Comment by alloy** [comment]
> I've deleted that class (`AROnboardingMoreInfoViewController`) - it wasn't being used elsewhere.

AWESOME 👍

File: Artsy/Networking/ARAuthProviders.m [hunk 1]
```diff
@@ -58,7 +58,7 @@ + (void)getTokenForFacebook:(void (^)(NSString *token, NSString *email, NSString
           failure(error);
 
         } else {
-            FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:@"me" parameters:nil];
+            FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:@"me?fields=name,id,email" parameters:nil];
```
Comment (mennenia) [inline Artsy/Networking/ARAuthProviders.m]: Doesn't seem like it - this confused me also and it's what I spent the longest on.

So, all the docs tell you in [this](https://developers.facebook.com/docs/ios/graph) (under "Fetch User Data") - I couldn't find the format in which one would request `email` in the parameters. I tried a dictionary with fields, an item `fields`, with an array, a bunch of options. 

The docs seemed to suggest that they expect a proper graph path, and trying out their [interactive one](https://developers.facebook.com/tools/explorer/), this path was the only one that gave us the data. I decided to try it in our code, and it was the one thing that worked.

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Needs testing on a Phone if change is non-trivial</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |251.1ms | [AuctionViewController.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Auction/AuctionViewController.swift#L199) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
177.7ms | [SwiftyJSON.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
161.0ms | [WebSocket.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
84.9ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L47) | @objc override func viewDidLoad()
73.5ms | [Observable.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
70.2ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, salesPerson: LiveAuctionsSalesPersonType, sideMargin: String)
68.5ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
61.6ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
59.7ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/mc-facebooklogin-improved/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by alloy** [comment]
Yup, seems perfect to me, rocking 👍

**Pull request merged**